src/main/java/Duke.java

```diff
@@ -1,10 +1,90 @@
+import java.util.Scanner;
+
 public class Duke {
     public static void main(String[] args) {
-        String logo = " ____        _        \n"
-                + "|  _ \\ _   _| | _____ \n"
-                + "| | | | | | | |/ / _ \\\n"
-                + "| |_| | |_| |   <  __/\n"
-                + "|____/ \\__,_|_|\\_\\___|\n";
-        System.out.println("Hello from\n" + logo);
+        String line = "____________________________________________________________\n";
+        Scanner cmd = new Scanner(System.in);
+        Task[] list = new Task[100];
+        int cnt = 0;
+        Boolean isLast = false;
+
+        System.out.println(line + " Hello! I'm Tommy\n" + " What can I do for you?\n" + line);
+
+        do {
+            String foo = cmd.nextLine();
+            String[] arr = foo.split(" ",2);
```

Comment:
The variable name arr does not explain what the array is being used for. Maybe you could rename this variable?